server/controllers/users.js

```diff
@@ -53,12 +53,12 @@ export default class userController {
           }
           return res.status(401).json({
             succes: false,
-            errors
+            errors,
```

Comment:
You are not returning a message. More so, you are returning raw unprocessed errors to the user. That's not fair to the user